content/sensu-go/6.8/catalog/catalog-reference.md

```diff
@@ -0,0 +1,1290 @@
+required     | false
+type         | 
+example      | {{< code yml >}}
+value: [[ subscriptions ]]
+{{< /code >}}
+
+##### Resource attributes
+
+api_version  | 
+-------------|------ 
+description  | Sensu API group and version for the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+api_version: core/v2
+{{< /code >}}
+
+name         | 
+-------------|------ 
+description  | Name of the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+name: nginx-metrics
+{{< /code >}}
+
+type         | 
+-------------|------ 
+description  | Type of the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+type: CheckConfig
+{{< /code >}}
+
+## Resource limits
+
+There is no limit on the number of resources you can bundle into a single integration.
+Each integration can include as many checks, event filters, handlers, and pipelines as you need to achieve your observability goals.
+For example, you can develop a single host monitoring integration that installs all of the checks you want to run on every server.
+
+## Check guidelines
+
+For integrations that create checks, list the resource definitions in your `sensu-resources.yaml` file in the following order:
+
+1. CheckConfig
+2. HookConfig
+3. Secret
+4. Asset
+
+The `sensu-integration.yaml` file for check resources should include at least one subscription, whether it is provided as a default or requested with a prompt.
+Subscriptions should be named according to the check's function.
+For example, a PostgreSQL monitoring check could include a subscription named `postgresql`.
+
+Use the YAML `>-` multiline block scalar syntax to wrap the check `command` value and make it easier to read.
+For example:
+
+{{< code yml >}}
+spec:
+  command: >-
+    check-disk-usage.rb
+    -w {{ .annotations.disk_usage_warning | default 85 }}
+    -c {{ .annotations.disk_usage_critical | default 95 }}
+{{< /code >}}
+
+Use tunables like [tokens][25] in your check commands as needed, sourced from entity annotations (not labels) and with explicitly configured default values.
+
+Check resources should use [interval scheduling][26] with a minimum interval of 30 seconds.
+Set check `timeout` to a non-zero value that is no greater than 50% of the interval.
+
+Prompts for check pipelines should use one of the following generic categories:
+
+- Alerts
+- Incident management
+- Metrics
+- Events
+- Deregistration
+- Remediation
+
+## Pipeline guidelines
+
+For integrations that create pipelines, list the resource definitions in your `sensu-resources.yaml` file in the following order:
+
+1. Pipeline
+2. Handler, SumoLogicMetricsHandler, and TCPStreamHandler
+3. Filter
+4. Mutator
+5. Secret
+6. Asset
+
+For alert and incident management pipelines, do not use event filters that have subjective configuration options.
+Use the built-in [is_incident][27] and [not_silenced][28] filters.
```

Comment:
```suggestion
For alert and incident management pipelines, we recommend using the built-in [is_incident][27] and [not_silenced][28] event filters instead of custom event filters that are configured for specific use cases.
```